L/libjulia/common.jl

```diff
@@ -478,7 +480,7 @@ function build_julia(ARGS, version::VersionNumber; jllversion=version)
         push!(dependencies, BuildDependency("Zstd_jll")),
         push!(dependencies, BuildDependency(get_addable_spec("SuiteSparse_jll", v"7.10.1+0")))
         push!(dependencies, Dependency(get_addable_spec("LibUV_jll", v"2.0.1+20")))
-        push!(dependencies, Dependency(get_addable_spec("LibUnwind_jll", v"1.8.1+2"); platforms=filter(!Sys.isapple, platforms)))
+        push!(dependencies, Dependency(get_addable_spec("LibUnwind_jll", v"1.8.2+0"); platforms=filter(!Sys.isapple, platforms)))
```

Comment:
We normally make sure this matches precisely with what is used in the corresponding Julia release resp. commit. Is that the case here? I ask because the same Julia commit seems to be used to build this as was used for the previous release. So at least one "got it wrong".

of course this is a rule that could be bent if necessary -- is it?